app/controllers/mixins/ems_common_angular.rb

```diff
@@ -431,6 +431,7 @@ def build_credentials(ems, mode)
          ems.supports_authentication?(:bearer) && !params[:default_password].blank?
         creds[:hawkular] = {:auth_key => params[:default_password], :userid => "_"}
         creds[:bearer] = {:auth_key => params[:default_password]}
+        creds.delete(:default)
```

Comment:
This feels strange to be a UI thing...it also lends itself to breaking UI provider agnosticism.  I would say that alternately we should not be creating a default auth at all if it's not used, but then again, the common practice is that default is the main connection...in this case what has been called "bearer" I believe.  cc @blomquisg @durandom Thoughts?